scripts/close-milestone

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -x -e -o pipefail -u
+
+milestone_title=$1
+
+indented_milestones_json_array=$(gh api -H "Accept: application/vnd.github+json" /repos/VirtusLab/git-machete-intellij-plugin/milestones)
+
+number=$( echo "$indented_milestones_json_array" | jq -c --arg TITLE $milestone_title '.[] | select(.title | contains($TITLE)) | .number')
+
```

Comment:
I'd also check for whether there're still any open PRs/issues in the milestone, and only close the milestone if there aren't any